src/Web/Avalonia.Web.Blazor/AvaloniaView.razor.cs

```diff
@@ -56,90 +56,91 @@ internal INativeControlHostImpl GetNativeControlHostImpl()
         {
             return _nativeControlHost ?? throw new InvalidOperationException("Blazor View wasn't initialized yet");
         }
-
-        private void OnTouchStart(TouchEventArgs e)
+                
+        private void OnTouchCancel(TouchEventArgs e)
         {
             foreach (var touch in e.ChangedTouches)
             {
-                _topLevelImpl.RawTouchEvent(RawPointerEventType.TouchBegin, new Point(touch.ClientX, touch.ClientY),
+                _topLevelImpl.RawTouchEvent(RawPointerEventType.TouchCancel, new Point(touch.ClientX, touch.ClientY),
                     GetModifiers(e), touch.Identifier);
             }
         }
 
-        private void OnTouchEnd(TouchEventArgs e)
+        private void OnTouchMove(TouchEventArgs e)
         {
             foreach (var touch in e.ChangedTouches)
             {
-                _topLevelImpl.RawTouchEvent(RawPointerEventType.TouchEnd, new Point(touch.ClientX, touch.ClientY),
+                _topLevelImpl.RawTouchEvent(RawPointerEventType.TouchUpdate, new Point(touch.ClientX, touch.ClientY),
                     GetModifiers(e), touch.Identifier);
             }
         }
 
-        private void OnTouchCancel(TouchEventArgs e)
+        private void OnPointerMove(Microsoft.AspNetCore.Components.Web.PointerEventArgs e)
         {
-            foreach (var touch in e.ChangedTouches)
+            if (e.PointerType == "mouse")
```

Comment:
Let's change here and in other places to this:
```suggestion
            if (e.PointerType != "touch")
```

Because I tested with my pen tablet, PointerType comes as "pen". And it's not handled with neither of these events when we filter only by mouse or touch.

Reply:
For now, it's fine if pen will be handled in the same way as a mouse. There is no better pen API atm.